pkg/action/action.go

```diff
@@ -82,6 +89,15 @@ type Configuration struct {
 	Log func(string, ...interface{})
 }
 
+// RESTClientGetter gets the rest client
+type RESTClientGetter interface {
+	ToRESTConfig() (*rest.Config, error)
+	ToDiscoveryClient() (discovery.CachedDiscoveryInterface, error)
+	ToRESTMapper() (meta.RESTMapper, error)
+}
+
+type debug func(format string, v ...interface{})
```

Comment:
Because this type is used in the exported `InitActionConfig`, it should be exported as well. We also might want a more specific name like `DebugLog`

Reply:
Changes Made